6-Typescript/server/tests/it/delete-todo.test.ts

```diff
@@ -1,33 +1,33 @@
-import { TestKit } from '../env/TestKit';
-import {createTodo} from '../env/random'
-import { ITodo } from '../../interfaces/ITodo';
-const testKit = new TestKit()
+import { TestKit } from "../env/TestKit";
+import { createTodo } from "../env/mockData";
+import { ITodo } from "../../interfaces/interfaces";
+const testKit = new TestKit();
 describe("DELETE todo by id /api/todos/:id ", () => {
-testKit.beforeAndAfter();
+    testKit.beforeAndAfter();
 
     describe("given no id of todo in db", () => {
         it("should return 404", async () => {
-            let todo: ITodo = createTodo()            
-            await testKit.mongoDriver.modelPostTodo(todo)
-            testKit.appDriver.setUserId(todo.userId)
-            todo.todoId = 'sdgsdf' 
-            try{
-                const response = await testKit.appDriver.deleteTodoById(todo.todoId) 
-            }
-            catch(err: any) {     
+            let todo: ITodo = createTodo();
+            await testKit.mongoDriver.modelPostTodo(todo);
+            testKit.appDriver.setUserId(todo.userId);
+            todo.todoId = "sdgsdf";
```

Comment:
it isn't part of the test. if you just need a random string, you can use `Chance` for it


in this kind of test - you technically just check that for `sdgsdf` - your system behaves as expected, not for every random string

Reply:
yea i missed that.
I created a mock data generator file to create those random data using chance.